src/openrct2/windows/_legacy.cpp

```diff
@@ -134,7 +134,7 @@ money32 place_provisional_track_piece(
         _unkF440C5.z = z;
         _unkF440C5.direction = trackDirection;
         _currentTrackSelectionFlags |= TRACK_SELECTION_FLAG_TRACK;
-        viewport_set_visibility((gTrackGroundFlags & TRACK_ELEMENT_LOCATION_IS_UNDERGROUND) ? 1 : 3);
+        viewport_set_visibility((0 & TRACK_ELEMENT_LOCATION_IS_UNDERGROUND) ? 1 : 3);
```

Comment:
`((0 & TRACK_ELEMENT_LOCATION_IS_UNDERGROUND) ? 1 : 3)` will always be `3`, so this not seem correct to me.

Reply:
Isn't this line for a maze, and mazes cannot ever be put underground?

Reply:
Yeah this can be always 3 as the maze does not/did not affect gTrackGroundFlags at all.